src/buildExec-get-files.ts

```diff
@@ -0,0 +1,7 @@
+import * as glob from "@actions/glob";
+
+// This is in a separate file so that it can be mocked in tests
+export const getFilesByGlobs = async (files: string) => {
+  const globber = await glob.create(files.split(",").join("\n"));
+  return await globber.glob();
+};
```

Comment:
I've created a separate file for the `getFilesByGlobs` function so that it can be mocked in tests. However, I guess it would be much cleaner if we managed to have it in the same file (`buildExec.ts`)...
What do you think about it?